### PR TITLE
10 user story 2 frontend

### PR DESCRIPTION
**Pull request by macben888**

### CODE REVIEW

File: frontend/src/components/forms/custom-text/CustomText.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {TextField} from "@mui/material";
+import {ITextFieldProps} from "../../../interfaces/INewItem";
+
+//component imports
```
Comment (Haferkorn) [inline frontend/src/components/forms/custom-text/CustomText.tsx:5]: if you don't import something you could leave them out :-) But in general nice structure for the imports

File: frontend/src/components/forms/order/Order.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+//component imports
```
Comment (Haferkorn) [inline frontend/src/components/forms/order/Order.tsx:3]: if you import anything  you can leave the comments out

File: frontend/src/components/forms/supplier/Supplier.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+//component imports
```
Comment (Haferkorn) [inline frontend/src/components/forms/supplier/Supplier.tsx:3]: as above

File: frontend/src/components/main-view/grid-view/GridView.tsx [hunk 4]
```diff
@@ -3,25 +3,23 @@ import {Grid} from "@mui/material";
 import Thumbnail from "./thumbnail/Thumbnail";
 import {IThumbnailData, Views} from "../../../interfaces/IThumbnailData";
 import Details from "./details/Details";
+import {newItemData} from "../helpers";
 
 //component imports
```
Comment (Haferkorn) [inline frontend/src/components/main-view/grid-view/GridView.tsx:8]: as above

**Review by Haferkorn — APPROVED** [review]

File: frontend/src/components/forms/custom-text/CustomText.tsx (hunk 1 above)
Comment (macben888) [reply frontend/src/components/forms/custom-text/CustomText.tsx:5]: they get autogenerated by the script I use to create my component files. I actually forgot to move the imports to the right location in this file though, I notice now ;-)

**Pull request merged**